app/models/ocelot/stanzas/ValueStanza.scala

```diff
@@ -18,26 +18,42 @@ package models.ocelot.stanzas
 
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
-import play.api.libs.json.Reads._
+import play.api.libs.json.{Reads, Writes, OWrites}
 
 case class Value(valueType: ValueType, label: String, value: String)
 
 object Value {
 
   implicit val reads: Reads[Value] =
-    ((__ \ "type").read[ValueType] and
+    (
+    	(__ \ "type").read[ValueType] and
       (__ \ "label").read[String] and
       (__ \ "value").read[String])(Value.apply _)
 
+  implicit val writes: Writes[Value] =
+    (
+    	(__ \ "type").write[ValueType] and
+      (__ \ "label").write[String] and
+      (__ \ "value").write[String] 
```

Comment:
IntelliJ indicates there is some trailing whitespace here.

Reply:
Ok will strip out